app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.kt

```diff
@@ -14,3 +15,55 @@ fun setProfileDataText(textView: TextView, timestamp: Long) {
   val time = sdf.format(Date(timestamp))
   textView.text = String.format(textView.context.getString(R.string.profile_edit_created, time))
 }
+
+@BindingAdapter("profile:lastVisited")
+fun setProfileLastVisitedText(textView: TextView, timestamp: Long) {
+  // TODO(#555): Create one central utility file from where we should access date format or even convert date timestamp to string from that file.
+  textView.text =
+    String.format(
+      textView.context.getString(R.string.profile_last_used) + " " + getTimeAgo(
+        timestamp,
+        textView.context
+      )
+    )
+}
+
+private const val SECOND_MILLIS = 1000
+private const val MINUTE_MILLIS = 60 * SECOND_MILLIS
+private const val HOUR_MILLIS = 60 * MINUTE_MILLIS
+private const val DAY_MILLIS = 24 * HOUR_MILLIS
+
+// TODO(#555): Shift this logic to central utility file for date-time related conversions.
+fun currentDate(): Date {
+  val calendar = Calendar.getInstance()
+  return calendar.time
+}
+
+fun getTimeAgo(time: Long, context: Context): String {
+  var time = time
+  if (time < 1000000000000L)
+  // If timestamp is given in seconds, convert that to milliseconds.
+    time *= 1000
+
+  val now = currentDate().time
+  if (time > now || time <= 0) return ""
+
+  val res = context.resources
+  val timeDifference = now - time
+  return when {
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Its already formatted.